source/arch/metadatastore-format.rst

```diff
@@ -161,7 +161,6 @@ above ``event`` is described by the ``event_descriptor`` ::
 
     {
         "uid" : <unique_id>,
-        "type" : "measure",
```

Comment:
I am also not convinced that 'type' makes sense as the name.  The model provides information about what kind of event it is in other ways (what db/collection it is for one thing), type shadows all the things, and does not provide a clear ordering.

I think we should adopt the NASA language here and call this 'data_level' or some such where 0 is raw data.